templates/home.php

```diff
@@ -3,7 +3,7 @@
 
 $blog_page_id = get_option('page_for_posts');
 
-WPUtil\Component::render('components/banners/banner-default', [
+get_template_part('components/banners/', 'banner-default', [
```

Comment:
@gravcmcguire this should look more like this:

`get_template_part('components/banners/banner', 'default', [ ...args ]`